src/AuraLockerModule.sol

```diff
@@ -99,7 +107,10 @@ contract AuraLockerModule is
 
     /// @notice The actual execution of the action determined by the `checkUpkeep` method (AURA locking)
     function performUpkeep(bytes calldata /* _performData */ ) external override onlyKeeper {
-        // if (!SAFE.isModuleEnabled(address(this))) revert ModuleNotEnabled();
+        if (!_isModuleEnabled()) revert ModuleNotEnabled();
+
+        (, uint256 unlockable,,) = AURA_LOCKER.lockedBalances(address(SAFE));
+        if (unlockable == 0) revert NothingToLock(block.timestamp);
```

Comment:
im guessing formally reverting it will prevent chainlink from making the actual call, and thus saving gas?